enterprise/server/invocation_stat_service/invocation_stat_service.go

```diff
@@ -22,12 +23,14 @@ import (
 type InvocationStatService struct {
 	env environment.Env
 	h   interfaces.DBHandle
+	ch  interfaces.ClickHouseDBHandle
```

Comment:
I'm not a big fan of these short names for struct members. `ch` is not very obvious. 
I realize there's already an existing member called `h`, but I don't think that's much better.
What we rename `ch` to `olapdb` and separately rename `h` to `db`?

Reply:
Done